packages/storybook/stories/MessageThread/snippets/CustomBlockedMessage.snippet.tsx

```diff
@@ -43,7 +43,7 @@ export const MessageThreadWithCustoBlockedmMessageContainerExample: () => JSX.El
     {
       messageType: 'blocked',
       senderId: 'user2',
-      warningText: false,
+      warningText: '',
       linkText: 'This is custom blocked content with hyperlink only (microsoft.com)',
```

Comment:
Should this be undefined?

Reply:
This should be empty string so it will only show the "linkText" hyperlink